src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -6163,6 +6163,33 @@ GUI_TEST_CLASS_DEFINITION(test_6809) {
     CHECK_SET_ERR(nameList3[16] == "Bicolorana_bicolor_EF540830", "The 17 sequence is incorrect");
     CHECK_SET_ERR(nameList3[17] == "Hetrodes_pupus_EF540832", "The 18 sequence is incorrect");
 }
+
+GUI_TEST_CLASS_DEFINITION(test_6816) {
+    //1. Open "_common_data/fasta/pcr_test.fa".
+    GTFileDialog::openFile(os, testDir + "_common_data/fasta", "pcr_test.fa");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
```

Comment:
Wait for the sequence view is active before clicking. See GTUtilsSequenceView::checkSequenceViewIsActive...